recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -112,10 +114,15 @@ def _configure_cmake(self):
         if self.settings.compiler == "Visual Studio":
             self._cmake.definitions["WITH_CRT_DLL"] = True # avoid replacing /MD by /MT in compiler flags
 
-        if hasattr(self, "settings_build") and tools.cross_building(self, skip_x64_x86=True):
-            # FIXME: Toolchain file should provide valid value here
-            if self.settings.os == "Macos" and self.settings.arch == "armv8":
-                self._cmake.definitions["CMAKE_SYSTEM_PROCESSOR"] = "aarch64"
+        self._cmake.definitions["CMAKE_MACOSX_BUNDLE"] = False # avoid configuration error if building for iOS/tvOS/watchOS
```

Comment:
easier than patching upstream CMakeLists

Reply:
yes, much more elegant solution!

Reply:
Actually, it's the solution of upstream: https://github.com/libjpeg-turbo/libjpeg-turbo/commit/84d6306f64afd189de148ff13895537a24a55dd3